frontend/src/lib/components/Link.tsx

```diff
@@ -16,7 +16,9 @@ export function Link({ to, preventClick = false, tag = 'a', ...props }: LinkProp
             return
         }
 
-        if (!props.target) {
+        const isExternalLink = typeof to === 'string' && to.startsWith('http')
```

Comment:
The `Link` component is only intended for internal links as far as I understand, otherwise classic HTML `a` should b used (less React workload this way)

Reply:
tbh I think it is worth change. Without it, we would need to update [places like this](https://github.com/PostHog/posthog/blob/master/frontend/src/lib/components/PayGatePage/PayGatePage.tsx#L35) to have a check based on the type of link (for cloud it is relative, self-hosted is external) which I think will be prone to error if we expect devs to remember that always.

Reply:
Hmm, I see. Okay, let's update this then. But in this case I suggest we also handle the `mailto:` case in addition to `https?:`/